lib/src/models/side_menu.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:mustang_core/mustang_core.dart';
+
+@appModel 
+class $SideMenu {
```

Comment:
Rename this to `AppMenu`. If we choose to place at the bottom/top/right at a later time, it makes sense to keep the name generic.